js2-mode.el

```diff
@@ -51,21 +51,18 @@
 
 ;;   (add-to-list 'auto-mode-alist '("\\.js\\'" . js2-mode))
 
-;; Alternatively, to install it as a minor mode just for JavaScript linting,
-;; you must add it to the appropriate major-mode hook.  Normally this would be:
+;; Alternatively, you can install it as a minor mode just for JavaScript linting
+;; and/or to use packages that integrate with it.  (Also, in Emacs 27, the new
+;; JSX features added in that release are only accessible within this minor
+;; mode.  Linting of JSX code is also likely to fail.)  To install it as a minor
+;; mode:
 
 ;;   (add-hook 'js-mode-hook 'js2-minor-mode)
 
 ;; You may also want to hook it in for shell scripts running via node.js:
 
 ;;   (add-to-list 'interpreter-mode-alist '("node" . js2-mode))
 
-;; Support for JSX is available via the derived mode `js2-jsx-mode'.  If you
```

Comment:
Can we really remove this part, keeping in mind that pre-Emacs 27 it is *the* way to go?

Thinking more about it, maybe that's also the section that the Emacs 27 JSX instructions should go to instead. The "install it as a minor mode" section above gets too bloated with that extra info. Here, though, you can enumerate the cases. Use Emacs 27? Minor mode is the way to go (a note about linting problems would also be more relevant here). Use Emacs 26 or earlier? Do the stuff below.

Reply:
Yeah.  The organization and wording needed more massaging.  I made this change.